docs/admin-guide/install-buildout.md

```diff
@@ -83,6 +91,7 @@ bin/buildout
 
 This may take a few minutes.
 
+When ever the buildout configuration has changed, re-run ./bin/buildout
```

Comment:
Change to active voice, because the configuration doesn't change on its own.

```suggestion
Whenever you change the Buildout configuration, run `./bin/buildout` again.
```